services/QuillLMS/app/models/classroom.rb

```diff
@@ -168,7 +168,7 @@ def hide_all_classroom_units
   end
 
   def with_students
-    attributes.merge({students: students})
+    attributes.merge({students: students.sort_by { |s| s.last_name }})
```

Comment:
The outermost hash can be removed, but also you can use pretzel colon shorthand for last_name:

`attributes.merge(students: students.sort_by(&:last_name))`